test/KoreBuild.FunctionalTests/SimpleRepoTests.cs

```diff
@@ -74,16 +74,20 @@ public void FullBuildCompletes()
                     Assert.Collection(pkg.Descendants("File"),
                         a =>
                         {
-                            Assert.Equal("tools/any/any/Newtonsoft.Json.dll", a.Attribute("Path")?.Value);
+                            Assert.Equal("tools/any/netcoreapp2.1/Newtonsoft.Json.dll", a.Attribute("Path")?.Value);
```

Comment:
Interesting that this should be a result, do we know why?

Reply:
I switched the tests from using a .nuspec to using `PackAsTool`. When the tests were written, the SDK did not yet support packing tool projects.